### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by 3TUSK**

### CODE REVIEW

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 1]
```diff
@@ -10,13 +10,13 @@ jei.tooltip.recipe.ore.dict=接受下列任何物品：%s
...
-jei.tooltip.cheat.mode.how.to.disable.no.hotkey=%s-click here to toggle it.
+jei.tooltip.cheat.mode.how.to.disable.hotkey=按 %s 以切换。
+jei.tooltip.cheat.mode.how.to.disable.no.hotkey=按住 %s 并点击以切换。
 jei.tooltip.recipe.by=该配方由 %s 添加
-jei.tooltip.recipe.id=Recipe ID: %s
-jei.tooltip.not.enough.space=There is not enough space to display the ingredient list here.
-jei.tooltip.ingredient.list.disabled=The JEI overlay is disabled.
-jei.tooltip.ingredient.list.disabled.how.to.fix=Press %s to enable it.
+jei.tooltip.recipe.id=配方 ID: %s
+jei.tooltip.not.enough.space=屏幕空间不足，无法显示 JEI 界面。
```
Comment (liach) [inline src/main/resources/assets/jei/lang/zh_cn.lang:17]: 可以说清是配方列表吗

File: src/main/resources/assets/jei/lang/zh_cn.lang [hunk 2]
```diff
@@ -84,10 +84,12 @@ config.jei.advanced.debugModeEnabled=调试模式
...
-config.jei.advanced.modNameFormat=模组名称格式
-config.jei.advanced.modNameFormat.comment=定义 JEI 界面的信息提示中的 Mod 名称格式。留空以禁用。
-config.jei.advanced.maxColumns=Max Width
-config.jei.advanced.maxColumns.comment=The maximum width of the ingredient list.
+config.jei.advanced.modNameFormat=模组名称样式
+config.jei.advanced.modNameFormat.comment=定义 JEI 界面的信息提示中的 Mod 名称样式。留空以禁用。
+config.jei.advanced.maxColumns=最大宽度
+config.jei.advanced.maxColumns.comment=输入物品列表的最大宽度。
+config.jei.advanced.maxRecipeGuiHeight=合成 Gui 最大高度
+config.jei.advanced.maxRecipeGuiHeight.comment=合成表查询 Gui 允许的最大高度。
```
Comment (liach) [inline src/main/resources/assets/jei/lang/zh_cn.lang]: GUI 全大写？graphical user interface

File: src/main/resources/assets/jei/lang/zh_cn.lang (hunk 1 above)
Comment (3TUSK) [reply src/main/resources/assets/jei/lang/zh_cn.lang:17]: `There is not enough space to display JEI here.`

**Review by liach — APPROVED** [review]

File: src/main/resources/assets/jei/lang/zh_cn.lang (hunk 1 above)
Comment (liach) [reply src/main/resources/assets/jei/lang/zh_cn.lang:17]: Hmm, how come I can no longer mark this as resolved?

**Comment by mezz** [comment]
Thank you!

**Pull request merged**